api/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Use the offical Golang image to create a build artifact.
 # This is based on Debian and sets the GOPATH to /go.
-FROM golang:1.12.13-alpine as builder
+FROM golang:1.13-alpine as builder
```

Comment:
Please be specific in the golang version, as Dynatrace oneagent does not support newer versions yet.
![Screenshot from 2020-04-03 12-22-36](https://user-images.githubusercontent.com/56065213/78350663-d7df2300-75a5-11ea-9a6d-a5d7569a13b6.png)
I think 1.13.7 should be fine.

Reply:
done